jaeger-thrift/src/main/java/io/jaegertracing/thrift/reporters/protocols/ThriftUdpTransport.java

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
Seems like something wrong here - this class was moved from jaeger-core, but looks like the old version is still there in this PR.